hazelcast/src/main/java/com/hazelcast/scheduledexecutor/impl/ScheduledRunnableAdapter.java

```diff
@@ -19,106 +19,83 @@
 import com.hazelcast.core.ManagedContext;
 import com.hazelcast.partition.PartitionAware;
 import com.hazelcast.instance.impl.Node;
-import com.hazelcast.nio.ObjectDataInput;
-import com.hazelcast.nio.ObjectDataOutput;
 import com.hazelcast.nio.serialization.IdentifiedDataSerializable;
 import com.hazelcast.scheduledexecutor.NamedTask;
 import com.hazelcast.scheduledexecutor.StatefulTask;
 import com.hazelcast.internal.services.NodeAware;
 
-import java.io.IOException;
 import java.util.Map;
 import java.util.concurrent.Callable;
 
-public class ScheduledRunnableAdapter<V>
+public class ScheduledRunnableAdapter<V> extends AbstractTaskDecorator<V>
         implements IdentifiedDataSerializable, Callable<V>, NodeAware, PartitionAware, NamedTask, StatefulTask {
```

Comment:
minor: could you cleanup the interfaces which are already inherited from`AbstractTaskDecorator` such as `IdentifiedDataSerializable`, `Callable<V>`

Reply:
removed